### DISCUSSION TITLE
Automatic recognition of patch names and edge conditions

### DISCUSSION THREAD

**Discussion by SergioP1975** [nextfoam/baram#142] [Ideas]
Hi, I have noted that for inlet and outlet patch names defined in BaramMesh, in BaramFlow are assigned inlet and outlet boundary conditions automaticly, that is great, can be added also for symmetry, wedge and maybe others?

Best regards, and thanks in advance

EDIT: The title should say "and boundary conditions".

**Comment by kdg5424** [comment]
Hello @SergioP1975. This is Donggyu from NEXTFOAM.
I checked that "sym, symmerty" don't work for this function.
I'll upload on development list and tell it to the developers.
Thank you for your suggestion.

Best regards

Donggyu Kim

**Reply by SergioP1975** [reply]
Thanks, those small features makes the model setup faster.
